packages/channels/botpress-channel-web/README.md

```diff
@@ -289,7 +289,8 @@ window.botpressWebChat.init({
   textColorOnForeground: '#ffffff', // Element text color (header, composer, button..)
   showUserName: false, // Whether or not to show the user's name
   showUserAvatar: false, // Whether or not to show the user's avatar
-  enableTranscriptDownload: false // Whether or not to show the transcript download button
+  enableTranscriptDownload: false, // Whether or not to show the transcript download button
+  customStylesheet: '' // Url to custom .css file
```

Comment:
@dmytropanontko , I think we need an example here. Could you prepare some custom css-example, that customizes all the classes available for customization? It'd be great to include it into README.